### ISSUE TITLE
Is anyone actively working on MPATH?

### ISSUE THREAD

**Issue by matentzn** [PaulNSchofield/mpath#10]
We need a general discussion on modelling pathological entities across OBO and in particular phenotype ontologies.

**Comment by PaulNSchofield** [comment]
Yes , but we have made very few changes since the ontology has been stable for many years. Happy to join in with modelling pathology but please recall that MPATH is focussed on histopathology, not gross pathology or clin path. Its applied in the post composition of pathological entities generally, so you will not find specific precomposed terms here as that would be impossible to deal with.

**Comment by matentzn** [comment]
Thank you @PaulNSchofield

Would you also provide clarification on the taxon-scope of MPATH? Metazoan, Mammal or, as the title suggests, mouse? I.e. can we, in principle, use MPATH pathological processes in the definition of cross metazoan phenotypes?

**Comment by PaulNSchofield** [comment]
So it is narrowly focussed on Human and Rodents; rat and mouse. It is however broadly applicable to all mammals, and much of it may be used for all vertebrates. Outside vertebrates certainly a lot of the high level terms might also be used but I would hesitate to say that it would be comprehensive for all metazoans; pathological processes in lower metazoans are really primitive. We consistently use MPATH for axiomatising human, mouse, and fish phenotypes but bear in mind that it is used combinatorially so classes are post-composed with the appropriate animal anatomy/CL class or Uberon, together with the MPATh term qualified with the pathology arm of PATO for macro and microscopic morphological and pathological process types. This is how it is used in IMPC for example.
